core/src/main/java/org/apache/accumulo/core/client/ZooKeeperInstance.java

```diff
@@ -112,7 +113,7 @@ public ZooKeeperInstance(String instanceName, String zooKeepers) {
    *          {@link ClientConfiguration} which extends Configuration with convenience methods
    *          specific to Accumulo.
    * @since 1.9.0
-   * @deprecated since 2.0.0; use {@link Connector#builder()} instead
+   * @deprecated since 2.0.0; use {@link Accumulo#newClient()} instead
```

Comment:
Probably do not need to add since whole class is deprecated.

Reply:
True but I was thinking having the link is helpful since we changed the way to connect and users may be used to this constructor.

Reply:
The link is nice.  If that is the reason for having it, then it would probably make sense to add that info to the other constructor in ZookeeperInstance.